src/routes/pm/BPay.tsx

```diff
@@ -39,7 +39,11 @@ export default function BPAyPage() {
       await utils.api.pm.bPay.addWallet(sessionToken, bpayAccountItems),
       E.match(
```

Comment:
Is it correct to send the full array of elements to the addWallet here?
I'am asking because I see that only the id of the first one is used in the url redirection.

Reply:
We must send all bpay account items.
The only one shown will be the first.
But all will be added.